docs/en/sql-reference/sql-functions/table-functions/files.md

```diff
@@ -365,9 +344,88 @@ When used with SELECT, FILES() returns the data in the file as a table.
+| lo_supplycost    | int              | YES  |
+| lo_tax           | int              | YES  |
+| lo_commitdate    | int              | YES  |
+| lo_shipmode      | varchar(1048576) | YES  |
++------------------+------------------+------+
+17 rows in set (0.05 sec)
+```
+
+## FILES() for unloading
+
+From v3.2.0 onwards, FILES() supports writing data into files in remote storage. You can use INSERT INTO FILES() to unload data from StarRocks to remote storage.
+
+### Syntax
+
+```SQL
+FILES( data_location , data_format [, StorageCredentialParams ] , unload_data_param )
+```
+
+### Parameters
+
+All parameters are in the `"key" = "value"` pairs.
+
+#### data_location
+
+See [FILES() for loading - Parameters - data_location](#data_location).
+
+#### data_format
+
+See [FILES() for loading - Parameters - data_format](#data_format).
+
+#### StorageCredentialParams
+
+See [FILES() for loading - Parameters - StorageCredentialParams](#storagecredentialparams).
+
+#### unload_data_param
+
+From v3.2 onwards, FILES() supports defining writable files in remote storage for data unloading.
+
+```sql
+-- Supported from v3.2 onwards.
+unload_data_param::=
+    "compression" = "<compression_method>",
+    "partition_by" = "<column_name> [, ...]",
+    "single" = { "true" | "false" } ,
+    "target_max_file_size" = "<int>"
+```
+
+| **Key**          | **Required** | **Description**                                              |
+| ---------------- | ------------ | ------------------------------------------------------------ |
+| compression      | Yes          | The compression method to use when unloading data. Valid values:<ul><li>`uncompressed`: No compression algorithm is used.</li><li>`gzip`: Use the gzip compression algorithm.</li><li>`snappy`: Use the SNAPPY compression algorithm.</li><li>`zstd`: Use the Zstd compression algorithm.</li><li>`lz4`: Use the LZ4 compression algorithm.</li></ul>                  |
+| partition_by     | No           | The list of columns that are used to partition data files into different storage paths. Multiple columns are separated by commas (,). FILES() extracts the key/value information of the specified columns and stores the data files under the storage paths featured with the extracted key/value pair. For further instructions, see Example 5. |
+| single           | No           | Whether to unload the data into a single file. Valid values:<ul><li>`true`: The data is stored in a single data file.</li><li>`false` (Default): The data is stored in multiple files if the amount of data unloaded exceeds 512 MB.</li></ul>                  |
+| target_max_file_size | No           | The best-effort maximum size of each file in the batch to be unloaded. Unit: Bytes. Default value: 1073741824 (1 GB). When the size of data to be unloaded exceeds this value, the data will be divided into multiple files, and the size of each file will not significantly exceed this value. Introduced in v3.2.7. |
```

Comment:
For `examples`, I recommend to add short meaning words for every exmaple.
`Example 1` -> ` Example 1: Query a Parquet file`

Reply:
And, put `Example 5: unloading data using INSERT INTO FILES` at last.